setup-scripts/skynet-nginx.conf

```
@@ -10,6 +10,11 @@ server {
 	listen [::]:443 ssl http2;
 	server_name siasky.net www.siasky.net; # replace with actual server names
 
+	# Enable the following line if you want to have auto uuid support. This
+	# means users are able to upload Skyfiles without having to provide a uuid
+	# themselves.
+	# rewrite ^/skynet/skyfile(\?.*)?$ /skynet/skyfile/$request_time$1;
```

This would match both `/skynet/skyfile` and `/skynet/skyfile/nsai90a0s2` - I think it would be safe to enable it by default if user posts to `/skynet/skyfile` directly but leave an option to manually set it if clients wants to.

Could you test this configuration? (the trailing slash is optional)
```suggestion
	# rewrite ^/skynet/skyfile/?$ /skynet/skyfile/$request_time$1;
```

---

Oof thanks for catching it. It was actually just a copy paste from old config.
It should  be `rewrite ^/skynet/skyfile/?$ /skynet/skyfile/$request_id$1;`